scripts/pipeline/orthology_benchmark.py

```diff
@@ -351,5 +391,4 @@ def calculate_goc_scores():
     prep_input_for_orth_tools(os.path.join(args.out_dir, args.species_set), args.orthology_input)
     print("Running orthology inference tools...")
     run_orthology_tools(args.orthology_input, args.orthofinder_params)
```

Comment:
Ideally we would know the exact path of the results directory so that we could pass this to `extract_orthologs`. The following suggestion (requiring `import datetime` and `import shlex`) would allow us to do this by: 1) checking that `--name` and `--output` parameters are not specified in `args.orthofinder_params`, since our approach assumes these are not set by the user; 2) setting an OrthoFinder results directory suffix based on an ISO 8601-formatted string of the current date and time; and 3) checking for a results directory uniquely matching the requested results directory name. This approach would necessitate user intervention in some situations, but in those cases the existing solution would arguably require careful monitoring anyway.
```suggestion
    for unsupported_flagset in [("-n", "--name"), ("-o", "--output")]:
        if any(x in unsupported_flagset for x in shlex.split(args.orthofinder_params)):
            raise RuntimeError(
                f"argument --orthofinder_params: '{'/'.join(unsupported_flagset)}' not supported")
    res_ts = datetime.datetime.now().strftime("%Y%m%dT%H%M%S")
    orthofinder_params = f"{args.orthofinder_params} --name {res_ts}"

    run_orthology_tools(args.orthology_input, orthofinder_params)

    res_path_spec = os.path.join(args.orthology_input, "OrthoFinder", f"Results_{res_ts}*")
    res_path, *clashing_paths = glob.glob(res_path_spec)
    if clashing_paths:
        raise RuntimeError(f"multiple directories match OrthoFinder results path spec: '{res_path_spec}'")
```